doc/serve.txt

```diff
@@ -0,0 +1,79 @@
+# Serve distillation.
+#
```

Comment:
Can you add a single line with a discription of what this is used for?

It's unclear to me (without any effort to try and find out) if these are interfaces that are part of this library or an external one.